languages/en_US.po

```diff
@@ -1968,6 +1970,160 @@ msgstr "Previous"
 msgid "下一篇"
 msgstr "Next"
 
+#: admin_style.php:128
+msgid "文章页"
+msgstr "Posts"
+
+#: admin_style.php:129
+msgid "文章列表"
+msgstr "Post List"
+
+#: admin_style.php:130
+msgid "标题栏"
+msgstr "Blog Title"
+
+#: admin_style.php:389
+msgid "自动隐藏标题栏"
+msgstr "Automatically hide the title"
+
+#: admin_style.php:395
+msgid "开启后，页面向下滚动时标题栏会向上隐藏，向上滚动时会出现。"
+msgstr ""
+"The title bar will automatically hide when scrolling down, "
+"it will reappear on scrolling up."
+
+#: admin_style.php:400
+msgid "标题栏显示内容"
+msgstr "Title bar displays"
+
+#: admin_style.php:426
+msgid "点击标题栏返回顶部"
+msgstr "Return to top on click"
+
+#: admin_style.php:432
+msgid "开启后，点击标题栏可以返回页面顶部。此设置影响所有页面。</strong>"
+msgstr "Click the title bar after opening to return to top."
+
+#: admin_style.php:502
+msgid "如使用自定义 API，你需要在括号内填写 API URL 并确保 API 返回 <code>{\"text\":\"句子\"}</code> 的 JSON 格式。<a href=\"https://doc.flyhigher.top/mdx/zh-CN/config/custom_api/\" target=\"_blank\">详细信息</a></code>"
+msgstr "If you are using a custom API, you'll need to fill-in the API URL in parentheses and make sure that the API returns the JSON format <code>{\"text\":\"quote\"}</code> <a href=\"https://doc.flyhigher.top/mdx/zh-CN/config/custom_api/\" target=\"_blank\">details</a></code>"
+
+#: admin_style.php:500
+msgid "使用一言 API（自定义）"
+msgstr "Use a quote API (custom)"
+
+#: admin_style.php:132
+msgid "页脚"
+msgstr "Footer"
+
+#: admin_style.php:133
+msgid "Touch Bar 图标"
+msgstr "Touch Bar Icon"
+
+#: admin_style.php:134
+msgid "杂项"
+msgstr "Misc"
+
+#: admin_fn.php:92
+msgid "文章"
+msgstr "Article"
+
+#: 
+msgid "首页推荐文章获取方式"
+msgstr "get recommended articles from"
```

Comment:
I'm thinking about whether `posts` or `articles` are more suitable in this context